sources/packages/backend/libs/test-utils/src/factories/institution-location.ts

```diff
@@ -45,6 +45,7 @@ export function createFakeInstitutionLocation(
       count: 4,
       upcase: true,
     });
+  institutionLocation.hasIntegration = options?.initialValue?.hasIntegration;
```

Comment:
did we miss this in the previous iterations? or why was this added now?

Reply:
It was not needed till now.